### PR TITLE
fix(proposals): perpetual proposed markets

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes N/A

# Description ℹ️

Proposed markets do not display the right product type. 

# Demo 📺

After: 
Screenshot 2023-11-16 at 14 03 50
Before: 
image

# Technical 👨‍🔧

We were still using `futureProduct` instead of `product` that should be deprecated.

### CODE REVIEW

File: libs/deal-ticket/src/utils/validate-expiration.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { t } from '@vegaprotocol/i18n';
 import type { Validate } from 'react-hook-form';
 
-export const validateExpiration: Validate<string | undefined> = (
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const validateExpiration: Validate<string | undefined, any> = (
```
Comment (MadalinaRaicu) [inline libs/deal-ticket/src/utils/validate-expiration.ts:5]:
```suggestion
export const validateExpiration: Validate<string | undefined> = (
```

File: libs/deal-ticket/src/utils/validate-expiration.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { t } from '@vegaprotocol/i18n';
 import type { Validate } from 'react-hook-form';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (MadalinaRaicu) [inline libs/deal-ticket/src/utils/validate-expiration.ts:4]:
```suggestion
```

**Review by jeremyletang — APPROVED** [review]

**Review by edd — APPROVED** [review]

**Review by bwallacee — APPROVED** [review]

**Comment by dalebennett1992** [comment]
Market displayed correctly when in a proposed state, the voting column also removed. Paired with @bwallacee , as he said in slack it will need some fixes to the automation when this goes into develop.

**Review by dalebennett1992 — APPROVED** [review]

**Comment by edd** [comment]
Did some manual exploratory testing on Governance & Explorer and haven't seen any issues

**Pull request merged**